### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by IbrohimRasulov**
### In this milestone, I have done:
✅  Add a new JavaScript file and import it as a module: it will contain methods related to the status updates (completed: true/false).
✅  Add event listener to the checkbox (change).
✅  Update items object's value for completed key upon user actions.
✅  Implement a function for the "Clear all completed" button.
✅  Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

File: src/clearTasks.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { displayTasksFromLS } from './add&delete'; // eslint-disable-line import/no-cycle
+
+const clearTask = (tasks, todoList) => {
+  document.querySelector('#clr-btn').addEventListener('click', () => {
+    for (let i = 0; i < tasks.length; i += 1) {
+      if ((tasks[i].completed) === true) {
+        tasks.splice(i, 1);
+      }
+    }
```
Comment (petudeveloper) [inline src/clearTasks.js]: You have done an excellent job doing a function for the "Clear all completed" button, but as the instructions say: you should implement it with the method `Array.filter()`. Please refactor your code so you use this method. `Array.filter()` is a method that will be very important for your future projects.

**Review by petudeveloper — CHANGES_REQUESTED** [review]
Hi @IbrohimRasulov, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

very close


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[OPTIONAL] Please add on your readme file instructions on how to clone your project `git clone https://github.com/IbrohimRasulov/ToDo-List-App.git`, doing this will make your repository look more professional and will help the more inexpert developers. 
**_Note: I add this comment here because the readme file has no changes on this pull request._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/add&delete.js [hunk 2]
```diff
@@ -30,8 +31,8 @@ const printTask = (task) => {
 const deleteTask = (li, tasks) => {
   li.children[3].addEventListener('click', () => {
     for (let i = 0; i < tasks.length; i += 1) {
-      if ((tasks[i].index) === (li.children[2].id)) {
-        tasks.splice(i, 1);
+      if ((tasks[i].index) === Number(li.children[2].id)) {
+        tasks.filter((task) => task.completed === true);
```
Comment (MrBrN197) [inline src/add&delete.js]: - [ ] Hello I think you made a mistake concerning what the last reviewer was saying. they were stating that you need to use the `filter` method for the clearCompleted function. Here you've used it in the `deleteTask` method. Please remove the filter from this line and replace it with what it had before.

File: src/clearTasks.js (hunk 1 above)
Comment (MrBrN197) [inline src/clearTasks.js]: - [ ] Please see my previous comment. this is where the reviewer wants you to use the `filter` function. inside the `clearTask` method.

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @IbrohimRasulov  👋 

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by oloomoses — APPROVED** [review]
Hi @IbrohimRasulov,
 
Well done on making the suggested changes.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**